### PR TITLE
fix the mixer config used by pilot e2e

### LINKED ISSUE #3138: Mixer logs overfilling with useless debug messages
This is everywhere in the logs, should be enabled only in verbose debug mode
2018-02-03T05:23:43.304232Z	debug	crd/store.go:136	Retrying to fetch config...

### PR DESCRIPTION
**Pull request by jmuk**
Mainly to fill out the CRDs used by mixer, and the defualt set
of configurations. They are coming from
install/kubernetes/template/istio-mixer.yaml.tmpl

Fixes #3138

### CODE REVIEW

**Comment by jmuk** [comment]
@andraxylia PTAL. I think this is basically a good thing to do.

File: pilot/test/integration/testdata/mixer.yaml.tmpl [hunk 1]
```diff
@@ -82,3 +97,905 @@ spec:
...
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: accesslog
+  namespace: {{.IstioNamespace}}
+spec:
+  severity: '"Default"'
+  timestamp: request.time
+  variables:
```
Comment (douglas-reid) [inline pilot/test/integration/testdata/mixer.yaml.tmpl]: @kyessenov should we add x-request-id in here somewhere to fix that other issue?

**Comment by jmuk** [comment]
rebased and added a new CRD `solarwinds` which was added meanwhile

**Comment by andraxylia** [comment]
e2e-smoke fails because of #3320

**Comment by andraxylia** [comment]
istio-pilot-e2e fails with auth, similar to many other PRs.

File: pilot/test/integration/testdata/mixer.yaml.tmpl [hunk 2]
```diff
@@ -82,3 +97,922 @@ spec:
...
+metadata:
+  name: apikeies.config.istio.io
+  labels:
+    package: apikey
+    istio: mixer-instance
+spec:
+  group: config.istio.io
+  names:
+    kind: apikey
+    plural: apikeies
```
Comment (andraxylia) [inline pilot/test/integration/testdata/mixer.yaml.tmpl]: Another PR went in the changes the plural to the correct name apikeys. Please update.

Comment (jmuk) [reply pilot/test/integration/testdata/mixer.yaml.tmpl]: IIRC k8s has some internal rules for pluralizing words, and the current mixer refers to those rules which results in this weird `apikeies`. `apikeys` might not work -- but I might be wrong (or some k8s behaviors might get fixed meanwhile). Did we confirm this works?
cc @mandarjog 
see also: https://github.com/istio/old_mixer_repo/pull/1167

Comment (jmuk) [reply pilot/test/integration/testdata/mixer.yaml.tmpl]: okay -- it seems some things get fixed meanwhile, seems working with the following versions:
```
% kubectl version
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.6", GitCommit:"6260bb08c46c31eea6cb538b34a9ceb3e406689c", GitTreeState:"clean", BuildDate:"2017-12-21T06:34:11Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.2-gke.1", GitCommit:"4ce7af72d8d343ea2f7680348852db641ff573af", GitTreeState:"clean", BuildDate:"2018-01-31T22:30:55Z", GoVersion:"go1.9.2b4", Compiler:"gc", Platform:"linux/amd64"}
```

will fix this manually (but still, we need to fix `mixer/cmd/mixs/cmd/crd.go` for pluralizing rules)

Comment (jmuk) [reply pilot/test/integration/testdata/mixer.yaml.tmpl]: done. PTAL.

**Comment by ldemailly** [comment]
good old #3037

/test istio-pilot-e2e

File: pilot/test/integration/testdata/mixer.yaml.tmpl (hunk 1 above)
Comment (ldemailly) [reply pilot/test/integration/testdata/mixer.yaml.tmpl]: what other issue?

**Pull request merged**